external/storm-solr/src/main/java/org/apache/storm/solr/schema/builder/RestJsonSchemaBuilder.java

```diff
@@ -0,0 +1,53 @@
+package org.apache.storm.solr.schema.builder;
+
+import com.google.gson.Gson;
+import org.apache.storm.solr.schema.Schema;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Scanner;
+
+/**
+ * Class that buils the {@link Schema} object from the JSON representation of the schema as returned by the
+ * URL of the form http://localhost:8983/solr/gettingstarted/schema/ . This particular URL returns the schema
+ * in JSON format for the gettingstarted example running locally.
+ * <p></p>
+ * Created by hlouro on 7/28/15.
+ */
+public class RestJsonSchemaBuilder implements SchemaBuilder {
+    private static final Logger logger = LoggerFactory.getLogger(RestJsonSchemaBuilder.class);
+    private Schema schema;
+
+
+    /** Urls with the form http://localhost:8983/solr/gettingstarted/schema/ returns the schema in JSON format */
+    public RestJsonSchemaBuilder(String solrHost, String solrPort, String collection) throws IOException {
+        this(new URL("http://" + solrHost + ":" + solrPort + "/solr/" + collection + "/schema/"));
```

Comment:
Does solr gives config options to rename "/solr/" part and also "/schema/" If so we shouldn't be hard coding those here instead give as an option with defaults.